### PR TITLE
Unit tests 

### PR DESCRIPTION
**Pull request by Jgmedina95**
Hello, @SamCox822 and @qcampbel. (whoever is reviewing haha).
So, as always, went a little over the board. I added the conftest.py file because our test file was getting too damm long. 

I made it such that the path registry, and some test files were centralized.
@SamCox822 , I THINK I went a little into what you did for testing the setup and run functionalities, but is fine. we don't have enough testing either way.

### CODE REVIEW

File: tests/test_fxns.py [hunk 1]
```diff
@@ -44,39 +43,39 @@ def fibronectin():
     return "fibronectin pdb"
 
 
-@pytest.fixture
-def get_registry():
-    return PathRegistry()
+# @pytest.fixture
```
Comment (SamCox822) [inline tests/test_fxns.py:46]: can you remove these, as they appear unnecessary here? (and everywhere else this is)

File: tests/test_fxns.py [hunk 2]
```diff
@@ -160,13 +159,15 @@ def test_find_png(vis_fxns):
...
-def test_create_notebook(path_to_cif, vis_fxns):
+def test_create_notebook(ALA_CIF_file, vis_fxns):
+    path_to_cif = ALA_CIF_file
     result = vis_fxns.create_notebook(path_to_cif)
     path_to_notebook = path_to_cif.split(".")[0] + "_vis.ipynb"
     os.remove(path_to_notebook)
     assert result == "Visualization Complete"
 
 
+@pytest.mark.skip(reason="This has been deprecated")
```
Comment (SamCox822) [inline tests/test_fxns.py:170]: Why are we skipping these now?

File: tests/test_fxns.py [hunk 3]
```diff
@@ -425,14 +449,15 @@ def mock_listdir(path):
...
+def path_registry_with_mocked_fs(get_registry):
     with patch("os.path.exists", side_effect=mock_exists):
         with patch("os.listdir", side_effect=mock_listdir):
-            registry = PathRegistry()
+            registry = get_registry("raw", False)
             registry.get_timestamp = lambda: "20240109"
             return registry
 
 
+@pytest.mark.skip(reason="This was moved to test_pathregistry.py")
```
Comment (SamCox822) [inline tests/test_fxns.py:460]: If this is the case, can you remove it here, rather than just skipping?

**Review by SamCox822 — CHANGES_REQUESTED** [review]
Thanks for adding these! Quick notes:
- rather than comment things out, pls just remove it (as it appears to be redundant in most of these cases)
- can we avoid skipping tests for now? if they need fixed, we can fix them & if we need to remove them, we should
- also can you fix the merge conflicts?

thanks!

File: tests/test_fxns.py (hunk 1 above)
Comment (Jgmedina95) [reply tests/test_fxns.py:46]: sure

File: tests/test_fxns.py (hunk 2 above)
Comment (Jgmedina95) [reply tests/test_fxns.py:170]: We dont use the SimulationFunctions class anymore since we move to multi-input tools :)

Comment (SamCox822) [reply tests/test_fxns.py:160]: then can we remove it>

Comment (Jgmedina95) [reply tests/test_fxns.py:160]: yes we can

File: tests/test_pathregistry.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import os
+
+import pytest
+
+
+@pytest.mark.parametrize("with_files, raw_or_clean", [("raw", False), ("raw", True)])
+def test_registry_init(get_registry, with_files, raw_or_clean):
+    # make the test directory the cwd
```
Comment (SamCox822) [inline tests/test_pathregistry.py:8]: can you remove this? thanks!

File: tests/test_setupandrun.py [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+            }}
+        }}
+        """.format(
+            pdb_id=pdb_id
+        ).strip()
+
+    return create_input
+
+
+# @pytest.fixture(scope="module")
```
Comment (SamCox822) [inline tests/test_setupandrun.py:64]: can you remove this too?

**Review by SamCox822 — APPROVED** [review]
it looks good to me - can you just remove the extra comments before merging in? thanks!

**Pull request merged**